invenio_stats/aggregations.py

```diff
@@ -336,7 +339,8 @@ def agg_iter(self, dt):
                 ignore_cache=True,
             )
             for aggregation in results.aggregations["terms"].buckets:
-                doc = aggregation.top_hit.hits.hits[0]["_source"]
+                doc = aggregation.top_hit.hits.hits[0]["_source"].to_dict()
+                aggregation = aggregation.to_dict()
```

Comment:
The issue linked in the commit won't happen with the code as is right now because we're only using "leaf node" values (integers, strings, etc.).
If one of the values we reuse should be a dictionary however, it would be a DSL object (e.g. `AttrList`) instead, and cause issues when trying to reindex.